src/main/python/ultimate_source_of_accounts/account_exporter.py

```diff
@@ -110,9 +109,14 @@ def set_sns_topic_policy(self, topic_arn):
                 "sns:Subscribe"
             ],
             "Principal": {
-                "AWS": self.allowed_aws_account_ids
+                "AWS": "*"
             },
-            "Resource": topic_arn
+            "Resource": topic_arn,
+            "Condition": {
+                "StringEquals": {
+                    "aws:PrincipalOrgID": "o-okjxq6yr2i"
```

Comment:
Are you sure you want to hardcore your OrgID in this open source project?

Reply:
No. I'll change this. 
thank you.